model/note/note.go

```diff
@@ -258,7 +259,7 @@ func UpdateTitle(inst *instance.Instance, file *vfs.FileDoc, title string) error
 // ApplySteps takes a note and some steps, and tries to apply them. It is an
 // all or nothing change: if there is one error, the note won't be changed.
 // TODO fetch last info for file (if debounce)
-func ApplySteps(inst *instance.Instance, file *vfs.FileDoc, steps []couchdb.JSONDoc) error {
+func ApplySteps(inst *instance.Instance, file *vfs.FileDoc, lastRev string, steps []couchdb.JSONDoc) error {
```

Comment:
We can have a conflict with another file with the same filename. But it is managed.

And, for the moment, we expect that the file content has not been modified by something else that the notes application.

Reply:
I was thinking about title updates conflicts, from the notes application.
Since we don't change the revision of the Document when we update the title, I wonder if we could have a conflict with 2 people updating the title at the same time.

Reply:
The last one wins. The title is not managed by ProseMirror, so we don't have fancy diff/patch mechanism and 2 people editing the title at the same time is an edge case not managed by the front app.